### PR TITLE
Menu: Mark menu components as internal

### PR DESCRIPTION
**Pull request by torkelo**
Feel these components are not stable enough for plugin usage yet

### CODE REVIEW

**Comment by leeoniya** [comment]
i wish this process was inverse, with explicit marking of published APIs. the way it works now seems like a minefield of accidental API contracts :(

**Comment by torkelo** [comment]
Yea me too

File: public/app/plugins/panel/timeseries/TimeSeriesPanel.tsx [hunk 1]
```diff
@@ -55,10 +55,8 @@ export const TimeSeriesPanel: React.FC<TimeSeriesPanelProps> = ({
       <TooltipPlugin mode={options.tooltipOptions.mode} timeZone={timeZone} />
       <ZoomPlugin onZoom={onChangeTimeRange} />
       <ContextMenuPlugin timeZone={timeZone} replaceVariables={replaceVariables} />
-      {data.annotations ? (
+      {data.annotations && (
         <ExemplarsPlugin exemplars={data.annotations} timeZone={timeZone} getFieldLinks={getFieldLinks} />
-      ) : (
-        <></>
       )}
       {data.annotations ? <AnnotationsPlugin annotations={data.annotations} timeZone={timeZone} /> : <></>}
```
Comment (dprokop) [inline public/app/plugins/panel/timeseries/TimeSeriesPanel.tsx:61]:
```suggestion
      {data.annotations && (
        <AnnotationsPlugin annotations={data.annotations} timeZone={timeZone} />
      )}
```

**Review by dprokop — APPROVED** [review]

File: public/app/plugins/panel/timeseries/TimeSeriesPanel.tsx (hunk 1 above)
Comment (torkelo) [reply public/app/plugins/panel/timeseries/TimeSeriesPanel.tsx:61]: this is the current change

Comment (torkelo) [reply public/app/plugins/panel/timeseries/TimeSeriesPanel.tsx:61]: ah, the same change or annotations plugin

**Pull request merged**